apps/src/lib/client/utils.rs

```diff
@@ -901,6 +901,18 @@ pub fn pk_to_tm_address(
     println!("{tm_addr}");
 }
 
+pub fn default_base_dir(
+    _global_args: args::Global,
+    _args: args::DefaultBaseDir,
+) {
+    println!(
+        "{}",
+        get_default_namada_folder().to_str().expect(
+            "expected a default namada folder to be possible to determine"
```

Comment:
<pedantic>Error message is a bit cryptic</pedantic>

Reply:
Yeah +1 I'd like to see something else